tests/backends/spark_backend/test_descriptions_spark.py

```diff
@@ -0,0 +1,439 @@
+        },
+        "bool_01": {
+            "n": 9,
+            "count": 9,
+            "cv": 0.9486832980505138,
+            "n_distinct": 2,
+            "iqr": 1.0,
+            "is_unique": False,
+            "mad": 0,
+            "max": 1,
+            "min": 0,
+            "n_missing": 0,
+            "p_missing": 0,
+            "p_distinct": 2 / 9,
+            "p_zeros": 4 / 9,
+            "sum": 5,
+        },
+        "bool_01_with_nan": {
+            "n": 9,
+            "count": 9,
+            "p_missing": 0.0,
+            "memory_size": None,
+            "n_distinct": 3,
+            "n_unique": 1,
+            "p_distinct": 0.3333333333333333,
+            "is_unique": False,
+            "p_unique": 0.1111111111111111,
+            "n_infinite": 0,
+            "p_infinite": 0.0,
+            "n_zeros": 4,
+            "p_zeros": 0.4444444444444444,
+            "n_negative": 0,
+            "p_negative": 0.0,
+            "monotonic": None,
+            "quantiles": {"5%": 0.0, "25%": 0.0, "50%": 0.0, "75%": 1.0, "95%": 1.0},
+            "mad": 0.0,
+            "min": 0.0,
+            "max": check_is_NaN,
+            "mean": check_is_NaN,
+            "std": check_is_NaN,
+            "variance": check_is_NaN,
+            "kurtosis": check_is_NaN,
+            "skewness": check_is_NaN,
+            "sum": check_is_NaN,
+            "range": check_is_NaN,
+            "iqr": 1.0,
+            "cv": check_is_NaN,
+            "chi_squared": None,
+            "histogram": None,
+        },
+        "list": {
+            "n": 9,
+            "count": 9,
+            "n_missing": 0,
+            "p_missing": 0,
+        },
+        "mixed": {
+            "n": 9,
+            "count": 9,
+            "n_missing": 0,
+            "p_missing": 0,
+        },
+    }
+
+
+@pytest.mark.sparktest
+@pytest.mark.parametrize(
+    "column",
+    [
+        "id",
+        "x",
+        "y",
+        "cat",
+        "s1",
+        "s2",
+        "somedate",
+        "bool_tf",
+        "bool_tf_with_nan",
+        "bool_01",
+        "bool_01_with_nan",
+        "list",
+        "mixed",
+    ],
+)
+def test_describe_spark_df(
+    column,
+    describe_data,
+    expected_results,
+    summarizer,
+    typeset,
+    spark_session,
+):
+    cfg = Settings()
+    cfg.vars.num.low_categorical_threshold = 0
+    cfg.infer_dtypes = False
+    cfg.correlations["pearson"].calculate = False
+    cfg.correlations["spearman"].calculate = False
+    cfg.correlations["kendall"].calculate = False
+    cfg.correlations["cramers"].calculate = False
+    cfg.correlations["phi_k"].calculate = False
+    cfg.interactions.continuous = False
+    cfg.missing_diagrams["bar"] = False
+    cfg.missing_diagrams["dendrogram"] = False
+    cfg.missing_diagrams["heatmap"] = False
+    cfg.missing_diagrams["matrix"] = False
+
+    # cfg["spark"]["quantile_error"].set(0)
+
+    spark = spark_session
+
+    if column == "mixed":
```

Comment:
This preprocessing should not be here. Either the profiler supports it, or not and errors should be raises.